docs/user/04-metrics.md

```diff
@@ -297,24 +309,23 @@ spec:
+| `prometheus.io/scrape` (mandatory) | `true`, `false` | None | Controls whether Prometheus automatically scrapes metrics from this target.                                                                                                                                                                                                                                                             |
 | `prometheus.io/port` (mandatory)   | `8080`, `9100` | None | Specifies the port where the metrics are exposed.                                                                                                                                                                                                                                                                                           |
 | `prometheus.io/path`               | `/metrics`, `/custom_metrics` | `/metrics` | Defines the HTTP path where Prometheus can find metrics data.                                                                                                                                                                                                                                                                               |
 | `prometheus.io/scheme`             | `http`, `https` | If Istio is active, `https` is supported; otherwise, only `http` is available. The default scheme is `http` unless an Istio sidecar is present, denoted by the label `security.istio.io/tlsMode=istio`, in which case `https` becomes the default. | Determines the protocol used for scraping metrics — either HTTPS with mTLS or plain HTTP. |
 
 > [!NOTE]
-> The agent can scrape endpoints even if the workload is a part of the Istio service mesh and accepts mTLS communication. However, there's a constraint: For scraping through HTTPS, Istio must configure the workload using 'STRICT' mTLS mode. Without 'STRICT' mTLS mode, you can set up scraping through HTTP by applying the `prometheus.io/scheme=http` annotation. For related troubleshooting, see [Log entry: Failed to scrape Prometheus endpoint](#log-entry-failed-to-scrape-prometheus-endpoint).
+> The agent can scrape endpoints even if the workload is a part of the Istio service mesh and accepts mTLS communication. However, there's a constraint: For scraping through HTTPS, Istio must configure the workload using 'STRICT' mTLS mode. Without 'STRICT' mTLS mode, you can set up scraping through HTTP by applying the annotation `prometheus.io/scheme=http`. For related troubleshooting, see [Log entry: Failed to scrape Prometheus endpoint](#log-entry-failed-to-scrape-prometheus-endpoint).
 
-### Step 5: Activate Runtime Metrics
+### 5. Activate Runtime Metrics
 
-To enable collection of runtime metrics, define a MetricPipeline that has the `runtime` section enabled as input:
+To enable collection of runtime metrics for your containers and Pods, define a MetricPipeline that has the `runtime` section enabled as input:
```

Comment:
[PR 1241](https://github.com/kyma-project/telemetry-manager/pull/1241/files?file-filters%5B%5D=.go&file-filters%5B%5D=.md&file-filters%5B%5D=.yaml&owned-by%5B%5D=NHingerl&show-viewed-files=true#)

removed "for your Pods" but I don't understand why. Shall we keep "for your containers and Pods" or delete it completely (also in Traces)?

Reply:
We will extend the feature and will collect more then just container metrics, also volumes and nodes. That's why I think Mostafa removed it. Maybe:
```suggestion
To enable collection of runtime metrics like for containers, define a MetricPipeline that has the `runtime` section enabled as input:
```

Reply:
Or we simply drop it and keep Mostafa's version?

Reply:
The sentence misses value somehow. In order to get X, do X. What X is gets explained further down only, maybe:
```suggestion
To enable collection of metrics more specific to the container runtime, define a MetricPipeline that has the `runtime` section enabled as input:
```